pkg/printers/internalversion/describe.go

```diff
@@ -2407,8 +2407,8 @@ func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string, desc
 
 func describeNodeResource(nodeNonTerminatedPodsList *api.PodList, node *api.Node, w PrefixWriter) error {
 	w.Write(LEVEL_0, "Non-terminated Pods:\t(%d in total)\n", len(nodeNonTerminatedPodsList.Items))
-	w.Write(LEVEL_1, "Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\n")
-	w.Write(LEVEL_1, "---------\t----\t\t------------\t----------\t---------------\t-------------\n")
+	w.Write(LEVEL_1, "Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\tGPU Requests\tGPU Limits\n")
```

Comment:
The document says it should be set by using limits, but actually k8s help to [complete the requests](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/defaults.go#L126) and give a guarantee of [check](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/validation/validation.go#L50)